web/src/app/util/debug.ts

```diff
@@ -1,12 +1,14 @@
-export function debug(args: any): void {
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export function debug(message?: any, ...optionalParams: any[]): void {
```

Comment:
`message` should always be a `string` here, shouldn't it? if `error` as an object is passed here instead of `error.message`, it probably shouldn't be